app/views/admin/organisations/index.html.erb

```diff
@@ -12,7 +12,7 @@
 
 <%= link_to 'GovWifi map', admin_govwifi_map_index_path, class: "govuk-link" %>
 
-<p><input type='text' id='filter-input' placeholder='Search by organisation name' class='govuk-input'></p>
+<p><input type="search" id="filter-input" placeholder="Search by organisation name" class="govuk-input"></p>
```

Comment:
`search` is non-standard. Suggest not using it

https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/search_event

Reply:
Old browsers will fall back to using a text field here.
Won't prevent them from using this functionality minus some nice features of the UI?

Notify are using this input as well.